uwmmss0348.xml

```diff
@@ -9,8 +9,8 @@
 			>uw-mil-uwmmss0348</eadid>
 		<filedesc>
 			<titlestmt>
-				<titleproper encodinganalog="245$a">Finding Aid of the Student Wisconsin Education Association
-					Records</titleproper>
+				<titleproper encodinganalog="245$a">Finding Aid of the Student Wisconsin Education
```

Comment:
Changed the title based on the self-description of the organization on their FB page. (See below for more on this.) See: DACS 2.6.3